src/GUCardList.js

```diff
@@ -1,31 +1,30 @@
-import React from 'react'
-import 'composited-card';
-import Container from 'react-bootstrap';
-import { ListGroup } from 'react-bootstrap';
-import { Card, CardGroup } from 'react-bootstrap';
-import Col from 'react-bootstrap/Col';
-import Row from 'react-bootstrap/Row';
+import 'composited-card'
+import PropTypes from 'prop-types'
+import { Card } from 'react-bootstrap'
+import Row from 'react-bootstrap/Row'
 
 export default function GUCardList({ cardList }) {
+  return (
+    <Row xs={2} md={4} className='g-4'>
+      {cardList.map((card, index) => (
+        <Card key={index} bg='dark' style={{ 'border-width': '0px' }}>
+          <composited-card
+            protoId={card.protoId}
+            quality={card.quality}
+            responsiveSrcsetSizes={card.responsiveSrcsetSizes}
+          ></composited-card>
+        </Card>
+      ))}
```

Comment:
You can use maps for this kind of structures, where based on an array of items, you can render as many items as the array lenght. It is a pretty common pattern. No need a `for` loop. Just a `map` and iterate for every item in the array.

A `map` basically converts your array items into whatever you want. So you are converting every item of `cardList` into a `Card` element